### ISSUE TITLE
Unable to find framework to drag and drop as described on the Wiki

### ISSUE THREAD

**Issue by andrespch** [SVGKit/SVGKit#414]
Hello, according to the Wiki one should open the file 'SVGKit-iOS.xcodeproj' and then on the project navigator expand the folder 'SVGKitFrameworks', then find the framework, drag it and drop it. The problem is I only see some headers inside this folder (see image), could anybody please give me some guidance on what I'm doing wrong? I'd really appreciate it.

Thanks 

<img width="258" alt="captura de pantalla 2016-01-21 a las 11 39 31 p m" src="https://cloud.githubusercontent.com/assets/4707783/12496822/41fb34da-c098-11e5-9380-a3dd6a3b4439.png">

**Comment by adamgit** [comment]
Looks to me like that instruction is wrong, ought to say:

"Expand the folder "Products", and in there will be an SVGKitFramework file
you can drag/drop in".

On my install, it's currently showing two files in that folder both of
which have the same name, are actually the same file, because Xcode thinks
of tvOS and iOS as separate - but I think it's correct that one file should
have both in, if all is working as intended?

**Comment by andrespch** [comment]
hey @adamgit, I just tried that but under "Products" all files are marked in red, so they
 don't exist, did you manage to install it? any comments are appreciated.
<img width="250" alt="captura de pantalla 2016-01-22 a las 1 54 31 a m" src="https://cloud.githubusercontent.com/assets/4707783/12499466/1dc8ec0c-c0ab-11e5-8f19-1cda53d65797.png">

**Comment by andrespch** [comment]
I just added an image to show you what I see @adamgit

**Comment by adamgit** [comment]
You have to build it at least once, or else there's nothing to use...

**Comment by andrespch** [comment]
I've built it several times (for all targets) and the only file that get's created in the products folder is libSVGKit-iOS.2.0.0.a.

I've also tried the method of copy and pasting the library and the usr folder to my project and then I get "CSSPrimitiveValue_ConfigurablePixelsPerInch.h file not found"

If I tried to install it via cocoapods as also described on the wiki I get "[!] Unable to find a specification for 'SVGKit'."

**Comment by adamgit** [comment]
So ... the Framework files in the Products folder are displayed in red
instead of black?

**Comment by andrespch** [comment]
Yeah, as the image I posted above. I managed to get it running by imitating the way the demo project imports the library, but any of the methods described on the Wiki were helpful for me.

**Comment by adamgit** [comment]
I have an idea ... there's a nasty bug in Xcode 6 onwards where the HARDWARE you've selected changes how Xcode runs teh build commands.

You might have to select in the build drop-down:

"SVGKitFramework-iOS > iOS Device"

instead of

"SVGKitFramework-iOS > iPhone 6s" (or whatever you had there)

Also - I have changed the instructions to fix where it SHOULD have said "products folder"

**Comment by alexandre-g** [comment]
"SVGKitFramework-iOS > iOS Device" or in latest xcode "Generic iOS Device" would have fixed it. Can probably close this now..

**Comment by chriskaukis** [comment]
Wanted to mention I also had this problem. I did exactly as alexandre-g says. I've attached screenshots because it's easy to miss exactly what is meant.

<img width="532" alt="screen shot 2016-02-16 at 5 02 33 pm" src="https://cloud.githubusercontent.com/assets/455/13094570/217323ec-d4d1-11e5-9b81-bd65cc800fc5.png">

<img width="525" alt="screen shot 2016-02-16 at 5 02 48 pm" src="https://cloud.githubusercontent.com/assets/455/13094571/21854270-d4d1-11e5-9d2a-87b33db3071f.png">

**Comment by wendhsz** [comment]
I have the same issue but changing this SVGKitFramework-iOS > iOS Device didn't help me, also it's not the .a file that gets to exist when build but the framework. Also when I have this in my build "SVGKitFramework-iOS > iOS Device" xcode is asking me for an apple account, I got it working but the files are still not existing. Please help, my xcode version is 7.3 :(

**Comment by adamgit** [comment]
Might be a problem new to this version of xcode?

**Comment by wendhsz** [comment]
might be, anyone here experienced the same problem? I need help :(

**Comment by adamgit** [comment]
Sorry I can't help more, I'm on Xcode 7.0.x at the moment, none of our
projects have upgraded yet.

**Comment by alexandre-g** [comment]
@janefab20 http://www.goloskok.com/u/cap_2016-05-02_10-45-45_PM2.mov
Done on fresh clone of SVGKit 2.x branch with Xcode 7.3

If you want the .a file then select SVGKit-iOS as the target target with Generic iOS Device as device

**Comment by adamgit** [comment]
Great, thanks!

Is there something we can/should change in the default build settings that
would make this simpler?

e.g. is there something missing we should have configured to make
Frameworks build automatically? I didn't set it up, but when I looked
through it all looked correctly configured to me.

**Comment by alexandre-g** [comment]
I don't have much experience with neither frameworks nor static libs, but when trying to integrate this into the project as a framework I couldn't do it.. The framework I was getting was being built for arm only iirc, so could run on device but not on simulator. I've looked into it a little bit and from what I remember I understood - the framework was possibly done "the old way", and even though there are ways to get it to build for all architectures with build script sourcery, the new ios8 way of doing frameworks apparently does this automatically. Don't nearly have enough experience to vouch for this though.

As for static lib.. it was working, but I ended up importing the project as source files instead because I could not get it to build with bitcode which we wanted for the project

**Comment by adamgit** [comment]
Framework was definitely done the "new" way - it was created after iOS 8
shipped, and was done using the new Xcode features that only appeared with
iOS 8+.
